### PR TITLE
Corrected issue #22

### PR DESCRIPTION
**Pull request by PencilKnot**

### CODE REVIEW

File: days/day10.cpp [hunk 1]
```diff
@@ -1,36 +1,14 @@
 #include <iostream>
-#include "intro_line.h"
+#include "day10.h"
```
Comment (0foxes) [inline days/day10.cpp:2]: good, this includes the proper file

File: days/day10.cpp [hunk 2]
```diff
@@ -1,36 +1,14 @@
...
-        line += "";
-    } else if (day == 9){
-        line += "";
-    } else if (day == 10){
-        line += "Ten lords a-leaping\n";
-    } else if (day == 11){
-        line += "Ten lords a-leaping\n";
-    } else {
-        line += "Ten lords a-leaping\n";
+    while(day >= 10){
```
Comment (0foxes) [inline days/day10.cpp]: this line will just keep setting line while `day >= 10`, so it never stops

**Review by 0foxes — CHANGES_REQUESTED** [review]

**Review by wusteven815 — CHANGES_REQUESTED** [review]
Other than what @slightlyskepticalpotat wrote about `days/day10.cpp`, this pull request does not solve the second part of #22.

- Please rewrite using greater/less comparison operators (<, >, <=, >=)
- **Header file has a problem** (< this one)

**Comment by PencilKnot** [comment]
> Other than what @slightlyskepticalpotat wrote about `days/day10.cpp`, this pull request does not solve the second part of #22.
> 
> * Please rewrite using greater/less comparison operators (<, >, <=, >=)
> * **Header file has a problem** (< this one)

I'm a bit confused about what the problem is with the header file. I'm pretty sure I followed the exact format from _intro.cpp_ and I just changed the **#include "intro_line.h"** to **#include "day10.h"** to match the second day10 file.

**Comment by wusteven815** [comment]
> I'm a bit confused about what the problem is with the header file. I'm pretty sure I followed the exact format from intro.cpp and I just changed the #include "intro_line.h" to #include "day10.h" to match the second day10 file.

Your header file (`day10.h`) still has `intro_line.h` traces:
```c
#ifndef LEARNING_INTRO_LINE_H
#define LEARNING_INTRO_LINE_H
```

**Pull request merged**